src/resources/common.ts

```diff
@@ -108,33 +117,58 @@ export function standardContentManipulations($: any) {
   $('vimeo').remove();
   // $('cite').remove();
 
+  DOM.stripElement($, 'p ol');
+  DOM.stripElement($, 'p ul');
+  DOM.stripElement($, 'p li');
+  DOM.stripElement($, 'p ol');
+  DOM.stripElement($, 'p ul');
+  DOM.stripElement($, 'p li');
+  DOM.stripElement($, 'p ol');
```

Comment:
This repetition is intentional and necessary to handle multiply nested lists within paragraphs.